org.gaphor.Gaphor.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It should be put into upstream repo so other distributions can benefit too.

Reply:
Ok, removed, thanks :+1: